src/ui/server/helpers/is-production/index.test.ts

```diff
@@ -0,0 +1,33 @@
+import { isProduction } from '.';
+
+describe('helpers/isProduction', () => {
+  it('should return false when NODE_ENV is set to "development"', () => {
+    process.env.NODE_ENV = 'development';
+    expect(isProduction()).toBe(false);
+  });
+
+  it('should return false when NODE_ENV is set to "development"', () => {
+    process.env.NODE_ENV = 'staging';
+    expect(isProduction()).toBe(false);
+  });
+
+  it('should return false when NODE_ENV is not set', () => {
+    delete process.env.NODE_ENV;
+    expect(isProduction()).toBe(false);
+  });
+
+  it('should return false when NODE_ENV is set to an unexpected value', () => {
+    process.env.NODE_ENV = 'invalid';
+    expect(isProduction()).toBe(false);
+  });
+
+  it('should return false when NODE_ENV is set to an unexpected value', () => {
+    process.env.NODE_ENV = '';
+    expect(isProduction()).toBe(false);
+  });
+
+  it('should return false when NODE_ENV is set to an unexpected value', () => {
+    process.env.NODE_ENV = ' ';
+    expect(isProduction()).toBe(false);
+  });
```

Comment:
Not a blocker (in the interests of getting ready for PEN test) - duplicate tests here and the test description doesn't match the file path - I can pick this up, few improvements I have in mind can do in one PR 👍